apps/ui/src/components/Ui/Textarea.vue

```diff
@@ -35,17 +35,16 @@ watch(model, () => {
 
 <template>
   <UiWrapperInput
+    v-slot="{ id }"
```

Comment:
Why do we need named slots with `id`? I removed those and I don't see any change.